src/dfs/dfs_velocity.cpp

```diff
@@ -66,10 +66,12 @@ namespace SpherePoisson {
     {
         Int nrows = mat.extent(0);
         Int ncols = mat.extent(1);
+        Complex val;
+        Complex valb;
 
-        Kokkos::parallel_for("in",  Kokkos::MDRangePolicy<Kokkos::Rank<2>>({0,0}, {nrows, ncols}), KOKKOS_LAMBDA (const int i, const int j) {
-            Complex val = (str=="sin") ? Complex(0,0.5) : 0.5;
-            Complex valb = (str=="sin") ? -Complex(0,0.5) : 0.5;
+        val = (str=="sin") ? Complex(0,0.5) :Complex(0.5, 0.0);
```

Comment:
Just a heads up (we won't hold up the PR for this): `std::string` and cuda kernels don't play nice with each other.   If you want to eventually use a gpu for this, we'll have to either switch to a `char` array or define an `enum`.